guides/code-signing.md

```diff
@@ -4,20 +4,36 @@ Code signing is done for the Windows and Mac distributions of Cypress when they
 
 `electron-builder` handles code signing during the `create-build-artifacts` jobs. This guide assumes that the reader is already familiar with [`electron-builder`'s Code Signing documentation](https://www.electron.build/code-signing).
 
-## Installing a new Mac code signing key
+## Rotating the Mac code signing key
 
-Follow the directions supplied by `electron-builder`: https://www.electron.build/code-signing#travis-appveyor-and-other-ci-servers
+1. On a Mac, log in to Xcode using Cypress's Apple developer program identity.
+2. Follow Apple's [Create, export, and delete signing certificates](https://help.apple.com/xcode/mac/current/#/dev154b28f09) instructions:
+    1. Follow "View signing certificates".
+    2. Follow "Create a signing certificate", and choose the type of "Developer ID Application" when prompted.
+    3. Follow "Export a signing certificate". Set a strong passphrase when prompted, which will later become `CSC_KEY_PASSWORD`.
+3. Upload the exported, encrypted `.p12` file to the Cypress App Google Drive and obtain a public [direct download link][direct-download].
```

Comment:
Where does this go in the shared drive?

Reply:
Added direct links: [9296776](https://github.com/cypress-io/cypress/pull/25794/commits/9296776bdd254ed9c6dac77b8791da01613a6f68)